doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2472 @@
+  flat : numpy.flatiter object
+      Flattened version of the array as an iterator.  The iterator
+      allows assignments, e.g., ``x.flat = 3`` (See `ndarray.flat` for
+      assignment examples; TODO).
+  imag : ndarray
+      Imaginary part of the array.
+  real : ndarray
+      Real part of the array.
+  size : int
+      Number of elements in the array.
+  itemsize : int
+      The memory use of each array element in bytes.
+  nbytes : int
+      The total number of bytes required to store the array data,
+      i.e., ``itemsize * size``.
+  ndim : int
+      The array's number of dimensions.
+  shape : tuple of ints
+      Shape of the array.
+  strides : tuple of ints
+      The step-size required to move from one element to the next in
+      memory. For example, a contiguous ``(3, 4)`` array of type
+      ``int16`` in C-order has strides ``(8, 2)``.  This implies that
+      to move from element to element in memory requires jumps of 2 bytes.
+      To move from row-to-row, one needs to jump 8 bytes at a time
+      (``2 * 4``).
+  ctypes : ctypes object
+      Class containing properties of the array needed for interaction
+      with ctypes.
+  base : ndarray
+      If the array is a view into another array, that array is its `base`
+      (unless that array is also a view).  The `base` array is where the
+      array data is actually stored.
+
+  See Also
+  --------
+  array : Construct an array.
+  zeros : Create an array, each element of which is zero.
+  empty : Create an array, but leave its allocated memory unchanged (i.e.,
+          it contains "garbage").
+  dtype : Create a data-type.
+
+  Notes
+  -----
+  There are two modes of creating an array using ``__new__``:
+
+  1. If `buffer` is None, then only `shape`, `dtype`, and `order`
+     are used.
+  2. If `buffer` is an object exposing the buffer interface, then
+     all keywords are interpreted.
+
+  No ``__init__`` method is needed because the array is fully initialized
+  after the ``__new__`` method.
+
+  Examples
+  --------
+  These examples illustrate the low-level `ndarray` constructor.  Refer
+  to the `See Also` section above for easier ways of constructing an
+  ndarray.
+
+  First mode, `buffer` is None:
+
+  >>> np.ndarray(shape=(2,2), dtype=float, order='F')
+  array([[ -1.13698227e+002,   4.25087011e-303],
+         [  2.88528414e-306,   3.27025015e-309]])         #random
+
+  Second mode:
+
+  >>> np.ndarray((2,), buffer=np.array([1,2,3]),
+  ...            offset=np.int_().itemsize,
+  ...            dtype=int) # offset = 1*itemsize, i.e. skip first element
+  array([2, 3])
+
+This also works for functions and other objects that **you** create. Just remember to include a docstring with your function using a string literal (``""" """`` or ``''' '''`` around your documentation).
+
+For example, if you create this function:
+
+::
+
+  def double(a):
+    '''Return a * 2'''
+    return a * 2
+
+You can run:
+
+::
+
+  double?
+
+Which will return:
+
+::
```

Comment:
For brevity in the rst file, I can think you can combine these lines into
```
Which will return::
```

Reply:
yes that always works, would be nice indeed

Reply:
Sounds good!

Reply:
Got it